packages/@coorpacademy-components/.eslintrc

```diff
@@ -122,6 +122,14 @@
         "no-console": "off",
         "react/jsx-no-bind": "off"
       }
+    },
+    {
+      "files": [
+        "**/*.tsx"
+      ],
+      "rules": {
+        "react/prop-types": "off"
```

Comment:
disabling proptypes check for `.native` files since:
- they are typed here.
- their type is (must be) used by RN apps using them.

you're right though, I update the filter to ` "**/*.native.tsx"`

Reply:
I removed this override